### ISSUE TITLE
Font Issue?

### ISSUE THREAD

**Issue by thedotdesertfox** [malashin/hoi4treesnap#10]
![image](https://user-images.githubusercontent.com/87513689/187542923-61111e4c-beeb-446d-87bd-2f80c2e3797f.png)

I assume this is a font issue? do i have to just add in the font from vanilla to my mod?

**Comment by malashin** [comment]
My assumption is that this file inside your mod folder `\interface\core.gfx` contains the font declaration.
```
bitmapfont = {
	name = "hoi_16mbs"
#	path = "gfx/fonts/hoi_16mbs"
	fontfiles = {
		"gfx/fonts/hoi_16mbs"
		"gfx/fonts/hoi_16mbs_cryllic"
	}
	color = 0xffffffff

	textcolors = {
		G = { 86 172 91 }
		R = { 222 86 70 }
		Y = { 238 201 35 }
		O = { 238 123 35 }
		H = { 238 201 35 }
		T = { 255 255 255 }
		g = { 175 175 175 }
	}
}
```

Normally it is in `\interface\core.gfx` inside the game folder, and that font path is searched for in the game folder. But having it duplicated in the mod files makes this program look for the font file inside the mod folders `gfx/fonts/hoi_16mbs`, and your mod does not have that file in there.

You could remove `core.gfx` from you mod if you are not actually replacing anything in that file or add font files from the `gfx/fonts/` to your mod
```
hoi_16mbs.dds
hoi_16mbs.fnt
hoi_16mbs_cryllic.dds
hoi_16mbs_cryllic.fnt
```

Proper solution would be me adding a fallback to the game folders font if the font is not found in the mod folder.

**Comment by malashin** [comment]
Should be fixed in 0.5.12. Please download the new version.
https://github.com/malashin/hoi4treesnap/releases/tag/v0.5.12
